tests/openthread_test/Makefile

```diff
@@ -0,0 +1,47 @@
+APPLICATION = openthread_test
```

Comment:
I'll prepare something when I find the time :-). (I guess porting `posix_sockets` has precedence before that ;-))